pkg/controllers/restore/controller.go

```diff
@@ -220,24 +222,39 @@ func (h *handler) OnRestoreChange(_ string, restore *v1.Restore) (*v1.Restore, e
 		return h.setReconcilingCondition(restore, fmt.Errorf("error restoring CRDs, check logs for exact error"))
 	}
 
-	logrus.Infof("Starting to restore clusterscoped resources for restore CR %v", restore.Name)
-	// then restore clusterscoped resources, by first generating dependency graph for cluster scoped resources, and create from the graph
-	if err := h.restoreClusterScopedResources(ownerToDependentsList, &toRestore, numOwnerReferences, created, objFromBackupCR); err != nil {
+	var ownerToDependentsList map[string][]restoreObj
+	var toRestore []restoreObj
+
+	// then restore cluster-scoped resources, by first generating dependency graph for cluster scoped resources, and create from the graph
+	logrus.Infof("Starting to restore cluster-scoped resources for restore CR %v", restore.Name)
+	ownerToDependentsList = make(map[string][]restoreObj)
+	toRestore = []restoreObj{}
```

Comment:
We should basically never do var > map, this is a nil map and causes panics. `toRestore = []restoreObj{}` is the same as `var toRestore []restoreObj`. Move the instantiation if needed but keep it the same.